packages/gatsby/src/query/__tests__/data-tracking.js

```diff
@@ -229,7 +229,7 @@ const setup = async ({ restart = isFirstRun, clearCache = false } = {}) => {
     }
   })
 
-  const queryIds = queryUtil.calcInitialDirtyQueryIds(store.getState())
+  const queryIds = queryUtil.calcDirtyQueryIds(store.getState())
```

Comment:
This change shouldn't be needed given we have to preserve `calcInitialDirtyQueryIds`